### PR TITLE
Header ready with dummy props

### LINKED ISSUE #264: Store Page: Store Header

### PR DESCRIPTION
**Pull request by 4R53N**
Resolves #264

### CODE REVIEW

**Comment by leestanley** [comment]
@steets250 Can you do a quick runthru on this I'll set a reminder for me to review it tmmrw

**Comment by steets250** [comment]
@leestanley yeah I'll give it a review

File: src/components/StoreHeader/index.jsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+
+import './style.less';
+
+function numberWithCommas(x) {
```
Comment (leestanley) [inline src/components/StoreHeader/index.jsx:7]: We can probably move this to the utils file at `root/src/utils.js`

File: src/components/StoreHeader/index.jsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+StoreHeader.defaultProps = {
+  hideCart: true,
+};
+
+CartLink.propTypes = {
+  cartItems: PropTypes.number.isRequired,
+};
+
+export default StoreHeader;
+// TODO add proptypes
```
Comment (leestanley) [inline src/components/StoreHeader/index.jsx:56]: We needa move this to TypeScript

**Comment by leestanley** [comment]
> Looks good to me (besides previously mentioned need for Typescript). Component renders correctly on desktop + mobile.

Yup, I'm going to add a blocked tag on this until the TypeScript PR gets merged

**Comment by leestanley** [comment]
> > Looks good to me (besides previously mentioned need for Typescript). Component renders correctly on desktop + mobile.
> 
> Yup, I'm going to add a blocked tag on this until the TypeScript PR gets merged

@4R53N This should be resolved now, just update it in TS

File: src/containers/StoreHeader.tsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+const StoreHeaderContainer: React.FC<StoreHeaderContainerProps> = (props) => {
+  const history = useHistory();
+  function handleGoBack() {
+    history.goBack();
+  }
+  const { titleType, titleText, hideCart } = props;
+  return (
+    <StoreHeader
+      // points={props.points}
```
Comment (leestanley) [inline src/containers/StoreHeader.tsx:20]: Remove comments

File: src/containers/StoreHeader.tsx [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+      points={10000}
+      handleGoBack={handleGoBack}
+      // cartItems={props.cartItems}
```
Comment (leestanley) [inline src/containers/StoreHeader.tsx:23]: Same here

**Pull request merged**